src/wallet/walletdb.cpp

```diff
@@ -599,6 +599,16 @@ ReadKeyValue(CWallet* pwallet, CDataStream& ssKey, CDataStream& ssValue,
                 return false;
             }
         }
+        else if (strType == "hdchain")
```

Comment:
Which one? We did the redefine-magic-strings-as-constants it in other places, but also for the wallet?